src/cljs/rems/atoms.cljs

```diff
@@ -173,6 +174,34 @@
   [:div {:class "navbar-brand logo-menu"}
    [:div.img]])
 
+(defn expander
+  "Displays an expandable block of content with animated chevron.
+   
+   Pass a map of options with the following keys:
+   * `id` unique id for expanded content
+   * `content` content which is displayed in expanded state
+   * `expanded?` initial expanded state
+   * `title` content which is always displayed together with animated chevron"
+  [{:keys [id content expanded? title] :or {expanded? false}}]
+  (let [expanded (reagent/atom expanded?)
+        id (escape-element-id id)]
+    (fn []
```

Comment:
Shouldn't we repeat the parameters here?

Reply:
yes :+1:  minus the `expanded?` state which is only needed for the initial state